src/main/java/Assets/Asset.java

```diff
@@ -18,8 +19,9 @@ public Asset(double volume, double price, String type) {
     public Asset(double volume, double price, String type, String symbol) {
         this.type = type;
         this.volume = volume;
-        this.price = price;
         this.symbol = symbol;
+        this.initialPrice = price;
+        this.currentPrice = price;
```

Comment:
I think we only need one price which is the initialPrice in the Asset, can someone confirm it？

Reply:
I think both will work, we can just let assets store a current price but not use it. As long as they are able to change independently, I think it will be fine.